src/antlr/GroovyLexer.g4

```diff
@@ -882,7 +882,7 @@ ELVIS_ASSIGN    : '?=';
 
 // §3.8 Identifiers (must appear after all keywords in the grammar)
 CapitalizedIdentifier
-    :   [A-Z] JavaLetterOrDigit*
+    :   JavaLetter {Character.isUpperCase(getInputStream().LA(-1))}? JavaLetterOrDigit*
```

Comment:
`getInputStream().LA(-1)` could be simplified as  `_input.LA(-1)`

Reply:
LGTM. I'll merge and include this change.